sdv/constraints/tabular.py

```diff
@@ -1135,6 +1137,7 @@ class Unique(Constraint):
 
     def __init__(self, columns):
         self.columns = columns if isinstance(columns, list) else [columns]
+        self.constraint_columns = tuple(self.columns)
```

Comment:
Why do we need this?

Reply:
Because of this line in the validation: `if set(self.constraint_columns).issubset(table_data.columns.values):`.

If a constraint doesn't have the `constraint_columns` attribute, it won't get validated

Reply:
I see. I wonder why we even need that condition tbh

Reply:
It was there originally. I didn't remove it because the code would crash if a `constraint_column` is not in the `table_data.columns`, and it seems we want to do stuff with it in the function right after, `_validate_constraint_columns`.

Reply:
Ok. That's fine but in general we should try to decouple the validation from the stuff related to the columns model. Originally they were more tied together because we were only validating constraints with a transform, but since that is no longer the case, they don't need to be tied together. Every constraint needs to be validated now, whether or not it will even need the columns model.